tst/com/amazon/corretto/crypto/provider/test/AESGenerativeTest.java

```diff
@@ -63,9 +65,151 @@ public void teardown() {
     amzCipherDecrypt = null;
   }
 
+  @Test
+  public void testEncrypt91() throws Exception {
+    // The implementation of javax.crypto.CipherSpi::bufferCrypt in JDK 10
+    // has an issue that has been resolved in the following PR:
+    // https://github.com/openjdk/jdk/commit/c3a97b27e244f97c80a14388aea4fc425006a87e
+    // We disable this test for JDK 10.
+
+    // To replicate the error for other JDKs, copy-paste the buggy implementation in AesGcmSpi.java
+    // and run `./gradlew cmake_clean single_test
+    // -DSINGLE_TEST=com.amazon.corretto.crypto.provider.test.AESGenerativeTest`
```

Comment:
I see why you're doing this, but I'm usually against embedding large blocks of commented out code. Maybe you can document these repro steps in a ticket/issue or anywhere we might look if we decide to fix this issue?

Reply:
Since it's JDK 10, we won't fix this; having said that, I removed the commented block with the link to the buggy code.